code/game/gamemodes/endgame/xmas/krampus.dm

```diff
@@ -28,12 +28,60 @@
 	real_name = "Krampus"
 	status_flags = GODMODE|CANPUSH
 
-	New(var/new_loc)
-		h_style = "Bald"
-		..(new_loc, "Krampus")
-		maxHealth=999999
-		health=999999
+/mob/living/carbon/human/krampus/New(var/new_loc)
+	h_style = "Bald"
+	..(new_loc, "Krampus")
+	maxHealth=999999
+	health=999999
+	var/obj/item/weapon/krampus/sack = new /obj/item/weapon/krampus(src)
+	put_in_hands(sack)
 
+/mob/living/carbon/human/krampus/proc/SackEm(var/mob/M)
+	if(!istype(M))
+		return
+
+	var/datum/admins/Krampus = client.holder
+	if(!(Krampus && check_rights(R_BAN)))
+		return
+
+	var/response = alert("Ban them too, or just sack them?",,"Ban", "Sack", "Cancel")
+	if(response == "Cancel")
+		return
+	if(response == "Ban" && !M.ckey)
+		to_chat(src, "You can only do that to mobs with a ckey.")
+		return
+
+	M.drop_all()
+	M.loc = src //need somewhere to store them while they're getting banned
+	var/g
+	switch(M.gender)
+		if("male")
+			g = "boy"
+		if("female")
+			g = "girl"
+		else
+			g = "thing"
+	to_chat(world, "<span class='danger'>Krampus sacked [M]. What a naughty little [g].<span>")
+	log_admin("[key_name_admin(src)] sacked [key_name_admin(M)].")
+
+	if(response == "Ban")
+		Krampus.newban(M)
+
+	qdel(M)
+
+/obj/item/weapon/krampus
+	name = "Krampus' Sack"
+	desc = "Krampus' sack that he shoves naughty spacemen in."
+	icon = 'icons/mob/human_races/krampus.dmi' //you're holding a mini krampus :^)
```

Comment:
I could give this an actual icon_state but if I did I'd want to do it to krampus too and that's too much work.

Plus it's hilarious holding a tiny krampus.

![ejwnyfeogyamanc78a-0ndtqzracbe1uchrfy-4-_17e13z6avazq7axer8di9w-uag1x5jdqbwcobzjufqvh1vj2q986_a0bqmmgwhmmhamewobwtaigdalmnoczitoud4a9vwaxlul-f0biugjnw](https://cloud.githubusercontent.com/assets/4043940/20424362/5556c8da-ad2a-11e6-8b81-1afeb3fa5cff.png)